minimesos/src/main/java/com/containersol/minimesos/cluster/MesosContainer.java

```diff
@@ -12,4 +13,6 @@
 
     JSONObject getStateInfoJSON() throws UnirestException;
```

Comment:
After you introduced `State getState()`, is `getStateInfoJSON()` used anywhere besides implementations of `getState()`. If not, `getStateInfoJSON()` can be removed from the interface